testkit/package.json

```diff
@@ -31,13 +31,13 @@
   "main": "dist/index.js",
   "types": "dist/index.d.ts",
   "peerDependencies": {
-    "@kalix-io/kalix-javascript-sdk": "^1.0.0-M7"
+    "@kalix-io/kalix-javascript-sdk": "1.x"
   },
   "dependencies": {
     "testcontainers": "^8.10.1"
   },
   "devDependencies": {
-    "@kalix-io/kalix-javascript-sdk": "^1.0.0-M7",
+    "@kalix-io/kalix-javascript-sdk": "^1.0.0-M10",
```

Comment:
CI always installs a local snapshot for testing, and now for publishing. But given that we're using the package lock file, and `npm ci`, for other dependencies, this could be updated manually at times to a compatible version. Note that the semantic versioning probably doesn't support the `M{n}` format, but requires `m.{n}` or similar, with a dot-separated numeric part. But this should work better once we're past 1.0.0 final.